CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 
 ### Added
+- Change `setresuid` and `setresgid` functions to accept Option<Uid> instead of Uid
```

Comment:
Because it's a breaking change, this should go into the Changed section, not Added.